caom2-search-server/src/main/java/ca/nrc/cadc/search/cutout/stc/STCCutoutImpl.java

```diff
@@ -111,7 +111,8 @@ Region createRegion()
         else
         {
             final Shape position = getSpatialSearch().getPosition();
-            region = new ca.nrc.cadc.stc.Circle(Frame.ICRS, null, null,
+            // When formatted will generate a DALI string: "CIRCLE cval2 cval2 radius"
+            region = new ca.nrc.cadc.stc.Circle(null, null, null,
```

Comment:
Was the use of STC discussed and I missed it?  This could now be using the `ca.nrc.cadc.dali.Circle` instead.  This method could return a `ca.nrc.cadc.dali.Shape` and dispense with all of the obsolete STC stuff.

Reply:
removed use of stc files in favour of DALI.  This pushed the spectral and spatial cutout parameters into two separate value output from TAPSearcher - work that was going to be done in the next code step anyway.